data/json/items/tool/firefighting.json

```
@@ -220,6 +221,7 @@
     "to_hit": -1,
     "qualities": [ [ "PRY", 3 ], [ "HAMMER", 1 ], [ "DIG", 1 ] ],
     "use_action": [ "CROWBAR" ],
-    "melee_damage": { "bash": 15, "stab": 13 }
+    "melee_damage": { "bash": 15, "stab": 13 },
+    "weapon_category": [ "BATONS", "HOOKING_WEAPONRY" ]
```

```suggestion
    "weapon_category": [ "HOOKING_WEAPONRY" ]
```
Six times the weight of a baton.